recipes/pygsti/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyGSTi" %}
+{% set version = "0.9.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyGSTi-{{ version }}.tar.gz
+  sha256: 5352933469c6d356220642f9747181ba361c6fcaf47b156c5893380ad1f7c30d
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - cython
+    - numpy >=1.16.0
+    - pip
+  run:
+    - python
+    - scipy
+    - plotly
+    - pandas
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - pygsti
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://www.pygsti.info
+  summary: A python implementation of Gate Set Tomography
+  license: MIT AND Apache-2.0
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - basnijholt
+    - enielse
```

Comment:
@enielse could you verify you want to be a recipe maintainer? If no, I will remove you again ;)

Reply:
Did we get a response here?

Reply:
@ocefpaf, he verified this offline (a private message on Slack).

Reply:
We need it here. Sorry.

Reply:
I'm happy to be a maintainer, thanks!